CHANGELOG.md

```diff
@@ -2,7 +2,24 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## Platform Version 0.11.11 - UNRELEASED
+## Platform Version 0.11.11 - 2024-08-05
+
+### Added
+
+* Sdf-hub support updates ([#4124](https://github.com/infinyon/fluvio/issues/4124))
+
```

Comment:
Can you add a Line for #4122 here?  It looks like git cliff did not automatically pick up "feat: *" commits.

Reply:
Sure! added it

Reply:
Great, LGTM, ready to merge